### ISSUE TITLE
Proofreadpages <pages> tag passed on unknown attributes

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#635]
Looking at the page `fr.wikisource.org/wiki/Rose_et_Blanche` in xowa-gui
![frheads](https://user-images.githubusercontent.com/861631/70519568-10a94700-1b34-11ea-972a-eaa59bbc511c.jpg)
The header/summary appears more than once (in fact for this page it appears 5 times)

Looking at the wikitext of the first two \<pages entries

    <pages index="Sand - Rose et Blanche ou la comédienne et la religieuse, Tome I, 1831.djvu" header=sommaire volume="5 volumes"  />
    <pages index="Sand - Rose et Blanche ou la comédienne et la religieuse, Tome II, 1831.djvu" header_type=empty  />
Note, in the first line `volume=` and in the second line `header_type=`
Neither of these are keywords \<pages> knows about
However on a further inspection of the PHP code I noticed that unknown attributes are passed onto the template 

My attempt at a fix is in
[Pp_pages_nde.zip](https://github.com/gnosygnu/xowa/files/3944264/Pp_pages_nde.zip)

Basically, keep a list of unprocess attribs and when necessary add them back to MediaWiki:Proofreadpage_header_template

**Comment by gnosygnu** [comment]
Great find as well as the suggested commit! Resolved above. Thanks!

**Comment by gnosygnu** [comment]
Oops. Tagged to commit to wrong issue. Commit is now above!
